multi-int-store/src/main/java/com/connexta/multiintstore/controllers/StoreController.java

```diff
@@ -88,7 +88,7 @@ public StoreController(
       return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
     }
 
-    return ResponseEntity.created(location).build();
+    return ResponseEntity.created(location).body(location);
```

Comment:
Return the URI as a header and as a the body. Spring rest controllers will serialize the body back into a URI automatically.

Reply:
❗️Why was this change needed? The location is already returned as a header, which I believe is the recommended way to handle POST requests.